pkg/csi_driver/controller.go

```diff
@@ -222,7 +272,10 @@ func (s *controllerServer) generateNewFileInstance(name string, capBytes int64,
 			location = v
 		case paramNetwork:
 			network = v
-		// Unused
+
+		//Ignore the cidr flag as it is not passed to the cloud provider. It is only consumed by the controller to get an unreserved IP
```

Comment:
Mention in the comment that this argument will be processed later in `reserveIPRange`